app/src/main/AndroidManifest.xml

```diff
@@ -3,6 +3,9 @@
     xmlns:android="http://schemas.android.com/apk/res/android"
     xmlns:tools="http://schemas.android.com/tools">
 
+    <uses-permission android:name="android.permission.INTERNET"/>
```

Comment:
Internet permission is not needed to open a browser on the device, please remove this.